shpkpr/commands/cmd_config.py

```diff
@@ -53,16 +52,15 @@ def set(ctx, application_id, env_vars):
 def unset(ctx, application_id, keys):
     """Unset application configuration.
     """
-    application = ctx.marathon_client.get_application(application_id)
+    existing_application = ctx.marathon_client.get_application(application_id)
+    application = MarathonApplication({'id': application_id})
```

Comment:
What do you think about implementing `__copy__` or `__deepcopy__` methods for the application object returned from `get_application`? I'm thinking that there might eventually be more parameters that we would need copied before re-submitting an application that we'll need to worry about here.